### DISCUSSION TITLE
Guidance on DNS zones and domain configuration (example.com)

### DISCUSSION THREAD

**Discussion by ranger-zero** [GoogleCloudPlatform/cloud-foundation-fabric#2042] [Q&A]
I am working in the 2-networking-a-peering stage and I am looking to understand more about how to configure my company domain within the configuration. I see example.com used as the placeholder and this made me curious as to if I should change this and what the impacts might be. 

My current application is cloud only with no on-prem with Google Domains as registrar.

**Comment by juliocc** [comment]
FAST networking stages only deals with [private zones](https://cloud.google.com/dns/docs/dns-overview#private_zone). It seems that you're referring to [public zones](https://cloud.google.com/dns/docs/dns-overview#private_zone), these are not managed by FAST.

If you need this, you can easily add public zones to the networking stage with the [DNS module](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/tree/master/modules/dns#public-zone).
